### PR TITLE
import-dob-permits/violations

### PR DESCRIPTION
**Pull request by rdlester**

### CODE REVIEW

File: JPF/import-dob-permits.py [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+        }
+
+
+perm_dob_df_keep_cols = [
+        'BOROUGH',
+        'Bin #',
+        'House #',
+        'Street Name',
+        'Job #',
+        'Job doc. #',
```
Comment (rdlester) [inline JPF/import-dob-permits.py:89]: Frak these column names, seriously, wtf with the period and not capitalizing every word

**Comment by jpfreeley** [comment]
I abhor #DO.b data.

File: JPF/import-dob-permits.py (hunk 1 above)
Comment (chrishenry) [reply JPF/import-dob-permits.py:89]: Points for `frak`

File: JPF/import-dob-permits.py [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+    else:
+        log.info("DOB Permits exists, moving on...")
+
+    perm_dob_description = 'DOB Permits'
+    perm_dob_pickle = dob_permits_dir + '/df_dob_permit.pkl'
+    perm_dob_sep_char = ","
+    perm_dob_table_name = "dob_permits"
+    perm_dob_load_pickle = args.LOAD_PICKLE
+    perm_dob_save_pickle = args.SAVE_PICKLE
+    perm_dob_db_action = "replace"
```
Comment (chrishenry) [inline JPF/import-dob-permits.py]: We should use the `--update-or-replace-db` [flag](https://github.com/chrishenry/deltanyc-heatseek/blob/master/JPF/utils.py#L38)

**Review by chrishenry — COMMENTED** [review]
Other than the flag, lgtm.

File: JPF/import-dob-permits.py (hunk 2 above)
Comment (rdlester) [reply JPF/import-dob-permits.py]: Done.

**Comment by chrishenry** [comment]
👍

**Pull request merged**